composer.json

```diff
@@ -37,31 +37,31 @@
         "sonata-project/exporter": "^1.11 || ^2.0",
         "sonata-project/form-extensions": "^0.1.1 || ^1.4",
         "sonata-project/twig-extensions": "^0.1.1 || ^1.3",
-        "symfony/asset": "^4.4 || ^5.1",
-        "symfony/config": "^4.4 || ^5.1",
+        "symfony/asset": "^4.4 || ^5.2",
+        "symfony/config": "^4.4 || ^5.2",
         "symfony/console": "^4.4",
-        "symfony/dependency-injection": "^4.4.8 || ^5.1",
+        "symfony/dependency-injection": "^4.4.8 || ^5.2",
         "symfony/doctrine-bridge": "^4.4",
         "symfony/event-dispatcher": "^4.4",
         "symfony/event-dispatcher-contracts": "^1.1 || ^2.0",
-        "symfony/expression-language": "^4.4 || ^5.1",
-        "symfony/form": "^4.4.12",
+        "symfony/expression-language": "^4.4 || ^5.2",
+        "symfony/form": "^4.4.12 || ^5.2",
```

Comment:
I think this one is actually tricky with keeping BC. It was also discussed here: https://github.com/sonata-project/SonataAdminBundle/pull/6362 (was switched from 3.x to master)

Reply:
Sure, that's why this PR is kept as draft yet.